src/realm/sync/config.hpp

```diff
@@ -123,6 +123,7 @@ enum class ClientResyncMode : unsigned char {
     RecoverOrDiscard,
 };
 
+// Also update sync_test_utils.hpp when adding new values
```

Comment:
It might be worth doing something similar to what instructions.hpp does with `REALM_FOR_EACH_INSTRUCTION_TYPE` to avoid having to update multiple places?